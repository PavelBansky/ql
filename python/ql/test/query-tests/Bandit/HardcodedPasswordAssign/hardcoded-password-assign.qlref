Bandit/hardcoded-password-assign.ql