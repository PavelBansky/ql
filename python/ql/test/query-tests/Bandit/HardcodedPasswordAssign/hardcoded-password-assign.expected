| Test.py:23:1:23:18 | AssignStmt | Possible hardcoded password. |
| Test.py:26:1:26:25 | AssignStmt | Possible hardcoded password. |
| Test.py:28:1:28:52 | AssignStmt | Possible hardcoded password. |
| Test.py:29:1:29:17 | AssignStmt | Possible hardcoded password. |
